.github/workflows/e2e.yaml

```diff
@@ -92,6 +92,7 @@ jobs:
           echo "Sleeping for 10 seconds to allow traces to be collected"
           sleep 10
           kubectl cp -c filecp traces/test-opentelemetry-collector-0:tmp/trace.json ./.github/workflows/e2e/bats/traces-orig.json
+          cat ./.github/workflows/e2e/bats/traces-orig.json
```

Comment:
is it intentional?

Reply:
Yes. Otherwise, it's hard to get the traces out to examine and fix tests without running the whole CI job step-by-step locally(which I did, but figured to add it there for the next time).

Reply:
@rauno56 FYI you can use the `make e2e-test` to run the test locally.